### DISCUSSION TITLE
need sponsorship

### DISCUSSION THREAD

**Discussion by ba138** [community/community#76047] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

how to get sponsorship . I am new to community any one help me

**Comment by anjuan** [comment]
Hi @ba138, welcome to the community! I'm the Engineering Manager for GitHub Sponsors, and I'd love to learn more about your goals for getting sponsorships? What would sponsorships help you accomplish?

**Reply by ba138** [reply, marked as answer]
Thank you for the warm welcome! I'm thrilled to connect with the GitHub Sponsors community.

Sponsorships would immensely support my ongoing open-source projects, enabling me to dedicate more time and resources to their growth and maintenance. With increased sponsorship, I aim to enhance the features, documentation, and overall user experience of my projects. I'm committed to fostering a vibrant and collaborative developer community, and sponsorships would be instrumental in achieving this goal.

I appreciate the opportunity to share my aspirations, and I look forward to contributing positively to the GitHub Sponsors ecosystem.

Best regards,
@ba138

**Reply by radahugo7R** [reply]
Thank you for you sponsor i really tankfull

**Comment by hasnain7676** [comment]
hi basit would you like to collaborate?

**Reply by ba138** [reply]
it is my pleaser

**Comment by ba138** [comment]
Hi Hasnain7676

Thank you for reaching out! I'm definitely open to collaboration and sponsorship opportunities. Could you please provide more details about the collaboration you have in mind? I'd like to learn more about the project or sponsorship and how I can contribute or be involved.

Looking forward to hearing more about it.

Best regards,
Basit
